modules/quadbin/snowflake/quadbin.js

```diff
@@ -0,0 +1,68 @@
+function tileToQuadbin (tile) {
```

Comment:
Not sure if you want to keep this file. It seems like using the old codification. Also there are few console.logs at the end

Reply:
i forgot to remove it, it doesn't make sense in this module, I just got it from the BQ code